cloudproxy-ui/src/components/Logs.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    {{ data }}
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ListProxies.vue",
+  data() {
+    return {
+      data: {},
+    }
+  },
+  beforeMount(){
+    this.getLogs();
+  },
+  methods: {
+    async getLogs(){
+      const res = await fetch('http://localhost:8000/logs');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/claffin/cloudproxy/pullRequest?prid=7329682)
```suggestion
      const res = await fetch("http://localhost:8000/logs");
```